src/components/pages/roster-page/edit-roster-entry-dialog.tsx

```diff
@@ -125,14 +125,25 @@ export const EditRosterEntryDialog = (props: EditRosterEntryDialogProps) => {
           return false;
         }
 
+        const value = rosterEntry[column.name] as string;
+
         switch (column.type) {
           case ApiRosterColumnType.String:
           case ApiRosterColumnType.Date:
-          case ApiRosterColumnType.DateTime:
-            if ((rosterEntry[column.name] as string).length === 0) {
+          case ApiRosterColumnType.DateTime: {
+            if (value.length === 0 || value.trim().length === 0) {
               result = false;
+            } else if (column.name === 'edipi') {
+              const chars = value.split('');
```

Comment:
This can be validated with the regex `/^[0-9]{10}$/g`

Reply:
Ha! Yes it can, thanks!

Reply:
Done